ServerCore/Areas/Identity/Pages/Account/Login.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@page
+@model LoginModel
+
+@{
+    ViewData["Title"] = "Log in";
+}
+
+<h2>@ViewData["Title"]</h2>
+<div class="row">
+    <div class="col-md-6 col-md-offset-2">
+        <section>
+            <h4>Use another service to log in.</h4>
+            <hr />
+            @{
+                if ((Model.ExternalLogins?.Count ?? 0) == 0)
+                {
+                    <div>
+                        <p>
+                            Something went wrong! Please contact puzzhunt@microsoft.com to report this error.
+                        </p>
+                    </div>
+                }
+                else
+                {
+                    <form asp-page="./ExternalLogin" asp-route-returnUrl="@Model.ReturnUrl" method="post" class="form-horizontal">
+                        <div>
+                            <p>
+                                Please login using your Microsoft (non-employee) account to register for an event or view your events.
```

Comment:
Does employee just work? Also what does "view your events" mean?

Reply:
I have no way to test that, so you tell me. And I'll reword the second part.